templates/default/django_settings.py.erb

```diff
@@ -41,142 +43,82 @@ LANGUAGE_CODE = 'en-us'
 
 SITE_ID = 1
 
+#Google analytics ID. Enter the full id, as in: UA-xxxxxx-x 
+GOOGLE_ID = 'UA-9880071-1'
```

Comment:
@mathuin What value should this be set too? I can add it to the data-bag if we want to do that now.